build.xml

```diff
@@ -293,7 +293,7 @@
     <sequential>
       <javadoc destdir="@{destdir}" author="true" version="true" use="true"
         windowtitle="${ant.project.name} API" classpathref="cassandra.classpath"
-        bottom="Copyright &amp;copy; 2009-2021 The Apache Software Foundation"
+        bottom="???? Copyright &amp;copy; 2009-2021 The Apache Software Foundation ????"
```

Comment:
The ASF copyright is still valid.  Append like `All changes to the original code are Copyright DataStax, Inc.`